modules/behaviors/DataBehavior.js

```diff
@@ -1036,6 +1077,7 @@
     dispose: function() {
```

Comment:
so you have a public dispose exposed on your behavior? the `this.off` and `this.listenTo` are already being performed by the hidden __dispose. Why not just move the `this.privateCollection.dispose()` to the _dispose you already have?

Reply:
this is on the .data object, not on the DataBehavior itself.  The Data object starts on line 880.